cmd/deploy.go

```diff
@@ -421,6 +421,14 @@ For more options, run 'func deploy --help'`, err)
 	return f.Stamp()
 }
 
+type ErrEnvNotExist struct {
+	Name string
+}
+
+func (e ErrEnvNotExist) Error() string {
+	return fmt.Sprintf("environment variable %q does not exist", e.Name)
+}
```

Comment:
can we put this into `pkg/functions/errors.go` please to keep consistent with the place for error definitions.
Otherwise looks good to me